core/db/migrate/20160420044191_create_spree_wallet_payment_sources.rb

```diff
@@ -0,0 +1,23 @@
+class CreateSpreeWalletPaymentSources < ActiveRecord::Migration[4.2]
+  def change
+    create_table :spree_wallet_payment_sources do |t|
```

Comment:
Before this we could probably add `return if table_exists?(:spree_wallet_payment_sources)`